app/controllers/api/application_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# rubocop:disable ClassAndModuleChildren
+class API::ApplicationController < ApplicationController
+  include Rails::Pagination
+
+  prepend_before_action :ensure_download_repositories_flipper_is_enabled
```

Comment:
Needed to use `prepend_before_action` here because I wanted to run this action before we run `authenticate_user!`. If I use a regular `before_action` then if the user is not logged in they get a forbidden message from `authenticate_user!` even if the flipper is disabled. 

I presume we want to show the 404 message if the flipper is not enabled as if the api doesn't exist right?

Reply:
Makes sense :+1: